phpunit.xml.dist

```diff
@@ -1,12 +1,12 @@
 <phpunit
```

Comment:
Ah, I forgot to force the push after rebase. The spare file should be all taken care of.